core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-device-selector-seo/dot-device-selector-seo.component.html

```diff
@@ -1,3 +1,8 @@
+<div
+    class="dot-device-selector-mask"
+    *ngIf="deviceSelector.overlayVisible"
+    data-testId="selector-mask"></div>
+
 <p-overlayPanel
     #deviceSelector
     (onHide)="onHideDeviceSelector()"
```

Comment:
What about `dismissable ` property of `p-overlAyPanel` "Enables to hide the overlay when outside is clicked." not works?

Reply:
@oidacra  It works, but it doesn't capture click events within the iframe. That's why I implemented it this way.

Reply:
Sad!..

Reply:
True... saw that before :(